helpers/target.go

```diff
@@ -252,21 +254,25 @@ func (c *GitCreds) Password() string {
 // in this same package.
 //
 // Verifications made depend on the asset type:
-//    - IP: None.
-//    - IPRange: None.
-//    - Hostname: NS Lookup resolution.
-//    - WebAddress: HTTP GET request.
-//    - DomainName: NS Lookup checking SOA record.
-//    - AWSAccount: Assume Role.
-//    - DockerImage: Check image exists in registry.
-//    - GitRepository: Git ls-remote.
+//   - IP: None.
+//   - IPRange: None.
+//   - Hostname: NS Lookup resolution.
+//   - WebAddress: HTTP GET request.
+//   - DomainName: NS Lookup checking SOA record.
+//   - AWSAccount: Assume Role.
+//   - DockerImage: Check image exists in registry.
+//   - GitRepository: Git ls-remote.
 //
 // This function does not return any output related to the process in order to
 // verify the target's reachability. This output can be useful for some cases
 // in order to not repeat work in the check execution (e.g.: Obtaining the
 // Assume Role token). For this purpose other individual methods can be called
 // from this same package with further options for AWS, Docker and Git types.
```

Comment:
I would mention that the reachability test is skipped if the env var is set to false.

Reply:
I would also mention the same in the doc comments for the other IsXXReachable functions

Reply:
Good point. Fixed in ad8fd0f.